src/validation/oas-result.ts

```diff
@@ -29,7 +32,7 @@ export default class OASResult {
     let resultString = '';
 
     if (this.error) {
-      resultString += `${this.testName}: Skipped - ${this.error}\n`;
+      resultString += `${this.suiteId} ${this.testName}: Skipped - ${this.error}\n`;
```

Comment:
I think I'd prefer to see these in the opposite order, thinking that testName will usually be an API name and version. Then we'd see `facilities-v0 oas-ruleset`, `facilities-v0 positive`, `forms-v0 oas-ruleset`, `forms-v0 positive`. Curious what others think too

Reply:
Yeah, I think I like the order @christineyost presented here. Ordering by API - version makes more sense to me, because a team, or one of us, or governance will likely be working at the API - version level.